src/16c8e0.c

```diff
@@ -28,7 +40,7 @@ EvtSource BtlPutPartnerAway = {
         EVT_SETF(EVT_VAR(0), EVT_FIXED(1.0))
         EVT_LOOP(10)
             EVT_CALL(SetActorScale, 256, EVT_VAR(0), EVT_VAR(0), EVT_FIXED(1.0))
-            EVT_SUBF(EVT_VAR(0), EVT_FIXED(0.1005859375))
+            EVT_SUBF(EVT_VAR(0), EVT_FIXED(0.1))
```

Comment:
This was just a test. IMO the script should definitely be amended to pretty print these (without ever losing accuracy). I have code for doing this in Star Rod if it would help.

Reply:
Yes please